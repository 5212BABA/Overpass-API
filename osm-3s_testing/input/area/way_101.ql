way(101);node(area)(if:id()>=10100 && id()<=10199);out;
