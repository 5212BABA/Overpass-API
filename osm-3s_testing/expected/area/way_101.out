<?xml version="1.0" encoding="UTF-8"?>
<osm version="0.6" generator="Overpass API">
<note>The data included in this document is from www.openstreetmap.org. The data is made available under ODbL.</note>
<meta osm_base=""/>

  <node id="10101" lat="51.2500001" lon="7.1500001"/>
  <node id="10102" lat="51.2500001" lon="7.1510001"/>
  <node id="10103" lat="51.2510001" lon="7.1510001"/>
  <node id="10104" lat="51.2510001" lon="7.1500001"/>
  <node id="10113" lat="51.2500001" lon="7.1500002"/>
  <node id="10114" lat="51.2500002" lon="7.1500002"/>
  <node id="10115" lat="51.2500002" lon="7.1500001"/>
  <node id="10125" lat="51.2500002" lon="7.1510001"/>
  <node id="10126" lat="51.2500002" lon="7.1510000"/>
  <node id="10127" lat="51.2500001" lon="7.1510000"/>
  <node id="10131" lat="51.2510000" lon="7.1510001"/>
  <node id="10137" lat="51.2510001" lon="7.1510000"/>
  <node id="10138" lat="51.2510000" lon="7.1510000"/>
  <node id="10141" lat="51.2510000" lon="7.1500001"/>
  <node id="10142" lat="51.2510000" lon="7.1500002"/>
  <node id="10143" lat="51.2510001" lon="7.1500002"/>

</osm>
